tests/lib/tools/lxd-state

```diff
@@ -1,7 +1,30 @@
 #!/bin/bash -e
 
 show_help() {
-    echo "usage: lxd-state [-h] {undo-mount-changes} ..."
+    echo "usage: lxd-state undo-mount-changes"
+    echo "       lxd-state prepare-snap"
+}
+
+prepare_snap(){    
+    echo "lxd-state: installing lxd snap"
+    snap install lxd --channel="$LXD_SNAP_CHANNEL"
+    if tests.pkgs is-installed lxd; then
+        echo "lxd-state: remove any installed pkgs (some images carry them) to ensure we use the snap"
+        tests.pkgs remove lxd lxd-client
```

Comment:
This fails on distributions that do not have the lxd-client package

```
lxd-state: remove any installed pkgs (some images carry them) to ensure we use the snap
+ apt-get remove --yes lxd lxd-client
Reading package lists...
Building dependency tree...
Reading state information...
Package 'lxd' is not installed, so not removed
E: Unable to locate package lxd-client
```

Reply:
Nice, thanks for the review, the issue should be fixed now.